.coveragerc

```diff
@@ -106,6 +106,7 @@ omit =
     homeassistant/components/thermostat/honeywell.py
     homeassistant/components/thermostat/nest.py
     homeassistant/components/thermostat/radiotherm.py
+    homeassistant/components/thermostat/homematic.py
```

Comment:
No, just order it alphabetically.

On Tue, Nov 24, 2015, 02:04 goir <notifications@github.com> wrote:

> In .coveragerc
> <https://github.com/balloob/home-assistant/pull/657#discussion_r45715901>:
>
> > @@ -106,6 +106,7 @@ omit =
> >      homeassistant/components/thermostat/honeywell.py
> >      homeassistant/components/thermostat/nest.py
> >      homeassistant/components/thermostat/radiotherm.py
> > +    homeassistant/components/thermostat/homematic.py
>
> What you mean, add tests ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/balloob/home-assistant/pull/657/files#r45715901>.
>